docs/scos/dev/architecture/architecture-performance.md

```diff
@@ -0,0 +1,211 @@
+
+Affected features 
+
+- Dataimport
+- Publish and Synchronize
+
+You can find more details here [Data Importer Speed Optimization | Spryker Documentation ](https://docs.spryker.com/docs/scos/dev/data-import/202108.0/data-importer-speed-optimization.html)
+
+### Database Query Optimization
+Database queries are the most slow parts of each applications, they have different dependencies such as database engines, hardware, configurations and etc. Spryker prevents any database execution for popular endpoints like Home, PDP, Search but for some this is not avoidable like Cart or Checkout. There are several ways to make sure these endpoints are handling the database queries in the best performance
+
+- Cache the result for the duplicate queries 
+- Aggregate several queries to only one query when it’s possible
+- Change single inserts to bulk inserts
+- Break down heavy or slow queries to smaller queries and use PHP native functionalities for result calculations (like sorting, group by, filtering, validations, ….)
+
+## Feature Configurations
+Spryker has different features and several configurable modules which need to be adjusted correctly to have the best performance and smooth execution in the applications, let’s check the most important configurations together 
+
+### Publish and Synchronize
+This feature is one the important infrastructure part in Spryker and it’s important that the configurations are set correctly
+
+#### Multiple publisher queues
+Publishers are using queues to propagate the events and let the workers to consume them to provide necessary data for our frontend services. As Spryker uses RabbitMQ as a default option, it’s recommended to use multiple queues instead of one to spread the loads between different queues. your can find more information about multiple publisher queues here [Integrating multi-queue publish structure | Spryker Documentation ](
+https://docs.spryker.com/docs/scos/dev/technical-enhancement-integration-guides/integrating-multi-queue-publish-structure.html#set-up-multiple-publish-queue-structure)
+
+#### Workers
+Spryker default configuration comes with 1 worker per publisher queues, but this can be increased to the max number of CPUs for specific queue if other queues are not receiving any loads. e.g.
+
+```
+Publisher.ProductAbstract 10000 msg/minute (2 workers)
+Publisher.ProductConcrete 10000 msg/minute (2 workers)
+Publisher.Transaltion 10 msg/minute (1 worker)
+Publisher.Cms 5 msg/minute (1 worker)
+....
+-------------------------------------------------------
+CPU: 4
+```
+
+#### Chunk size
+Publishers are using different chunks to consume the messages from queues, the best number is very dependent on each entity and the hardware, but as a best practice we recommend to choose one of these numbers:
```

Comment:
*the best number* - the best number of what?

Reply:
I would rather say "optimal size of the chunk"  instead of "the bets number"